src/Amadeus/Client/Struct/Service/IntegratedPricing.php

```diff
@@ -155,6 +161,37 @@ protected function loadPricingOptions($options)
 
         return $priceOptions;
     }
+    
+    /**
+     * @param FareBasis[] $pricingsFareBasis
+     * @return PricingOptionGroup[]
+     */
+    protected static function makePricingOptionFareBasisOverride($pricingsFareBasis)
+    {
+    	$opt = [];
+    
+    	if ($pricingsFareBasis !== null) {
+    		foreach ($pricingsFareBasis as $pricingFareBasis) {
```

Comment:
I don't know what intendation you are using here, but intendation has to be 4 spaces according to the PSR-2 standard.

Reply:
Changes done as per PSR-2 intendation.